src/plugins/intel_cpu/tests/functional/utils/quantization_utils.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <unordered_map>
+#include "openvino/op/util/attr_types.hpp"
+
+namespace CPUTestUtils {
+
+struct QuantizationData {
+    float il;
+    float ih;
+    float ol;
+    float oh;
+    int levels = -1;  // -1 means that levels are not defined
+    bool isPerChannel = false;
```

Comment:
I am not sure that we need this field: if we need to have per channel quantization, we need to specify FQ ranges as vector, not float values. By il-ih-ol-oh vectors we can easily understand if there is per tensor or per channel quantization